packages/commonwealth/client/scripts/views/components/ReactionButton/CommentReactionButton.tsx

```diff
@@ -66,7 +66,10 @@ export const CommentReactionButton = ({
   const hasReacted = !!(comment.reactions || []).find(
     (x) => x?.author === activeAddress,
   );
-  const likes = comment.reactionWeightsSum || comment.reactions.length;
+  const reactionWeightsSum = comment.reactions.reduce(
+    (acc, curr) => acc + (curr.calculatedVotingWeight || 1),
+    0,
+  );
```

Comment:
`comment.reactionWeightsSum` is the ideal source of truth, but this should work too

Reply:
hm - in my test, when a regular user adds reaction then `comment.reactionWeightsSum` gives `null or 0`, so i added the `calculatedVotingWeight` calculation just to be safe